backend/src/main/scala/bloop/DependencyResolution.scala

```diff
@@ -62,6 +63,7 @@ object DependencyResolution {
     }
     val fetch = ResolutionProcess.fetch(repositories, Cache.default.fetch)
     val resolution = start.process.run(fetch).unsafeRun()
+    if (shouldReportErrors) reportErrors(resolution, logger)
```

Comment:
Tested with Metals and now we will show a warning like:

![warning](https://user-images.githubusercontent.com/3807253/61655657-4b166280-acbf-11e9-9f38-6b901da39337.png)